packages/eslint-config-compass/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@mongodb-js/eslint-config-compass",
+  "version": "0.0.1",
+  "description": "Shared Compass eslint configuration",
+  "license": "SSPL",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "package.json",
+    "README.md"
+  ],
+  "peerDependencies": {
+    "eslint": "^7.25.0"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.28.4",
+    "@typescript-eslint/parser": "^4.28.4",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-mocha": "^8.0.0",
+    "eslint-plugin-react": "^7.24.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/compass.git"
+  },
+  "homepage": "https://github.com/mongodb-js/compass"
```

We should either mark it as `private: true` or set public access for the package, otherwise initial publish will fail (learned that it will happen literally only today 😅)

```suggestion
  "homepage": "https://github.com/mongodb-js/compass",
  "publishConfig": {
    "access": "public"
  }
```

---

Yep set that to public now, good spot :D